inputgraph/loader.go

```diff
@@ -454,6 +458,64 @@ func (l *loader) handleArg(ctx context.Context, cmd spec.Command, isBase bool) e
 	return nil
 }
 
+func (l *loader) handleLet(ctx context.Context, cmd spec.Command) error {
+	var opts commandflag.LetOpts
+	argsCpy := flagutil.GetArgsCopy(cmd)
+	args, err := flagutil.ParseArgsCleaned("LET", &opts, argsCpy)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse LET args")
+	}
+	if len(args) != 3 {
+		return errors.New("failed to parse LET args")
+	}
+	key := args[0]
+	baseVal := args[2]
+	val, err := l.expandArgs(ctx, baseVal)
+	if err != nil {
+		return wrapError(err, cmd.SourceLocation, "failed to expand LET value %q", baseVal)
+	}
+
+	l.hasher.HashString(fmt.Sprintf("LET %s=%s", key, val))
+
+	_, _, err = l.varCollection.DeclareVar(key, variables.WithValue(val))
+	if err != nil {
+
+		if err != nil {
```

Comment:
you must really want to check err isn't null here :)

(there's a duplicate if above)